schedule/jeos/sle/jeos-container-engines_and_tools.yaml

```diff
@@ -0,0 +1,40 @@
+---
+description: 'Test suite verifies functionality and integration of Containers (contm) and Package Hub (phub) modules'
+name: 'jeos_base+contm+phub'
+conditional_schedule:
+    bootloader:
+        MACHINE:
+            'svirt-xen-pv':
+                - installation/bootloader_svirt
```

Comment:
I am not 100% sure but I think that https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/bootloader.pm suppose to cover all conditions below . @OleksandrOrlov  I remember you were working on combine all boot logic into single place . WDYT ?

Reply:
Yep, there is a test module which should make this.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/bootloader_start.pm

Reply:
> Yep, there is a test module which should make this.
> 
> https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/bootloader_start.pm

Cool, I have checked only qsf-u group.

* https://openqa.suse.de/tests/4296280
* https://openqa.suse.de/tests/4296291

as we have the followup ticket from booting modules, I would leave this change for now. Thanks for the pointer